source/deploy/server/kubernetes/deploy-k8s-aks.rst

```diff
@@ -86,16 +82,18 @@ The installation process includes deploying Mattermost and updating the server.
 
       kubectl -n mattermost-operator get ingress
 
-  b. Get the resulting IP address from the ``ADDRESS`` column, and use your domain registration service to create a DNS record.
-  c. You should be good to go.
+9. Use your IP address from the ``ADDRESS`` column, and create a DNS record in your domain registration service. 
+
+10. Access your working Mattermost installation at the URL you’ve determined in your DNS record.
 
-Learn more about managing your Mattermost server by visiting the :doc:`Managing Mattermost </manage/admin/server-maintenance>` documentation.
+Learn more about administrating your Mattermost server by visiting the :doc:`Administration Guide </source/guides/administration-guide>`.
 
-**Step 2: Upgrade Mattermost**
+Upgrade Mattermost via your AKS cluster
+---------------------------------------
 
 1. Visit the ``Extensions + Applications`` section of your AKS cluster where your Mattermost installation is deployed.
 2. You can enable minor version auto upgrades since these are not updating Mattermost version
-3. Expand the ``Configurarion Settings`` table and add the below configuration and the version you want to install as a value.
+3. Expand the ``Configuration Settings`` table and add the below configuration and the version you want to install as a value.
 
   .. code:: 
 
```

Comment:
I think this "Upgrade Mattermost via your AKS cluster" heading is missing somehow, at least in the preview environment? 
![image](https://github.com/user-attachments/assets/f41f3fa4-efb0-4baa-a409-868229e9c337)